server/routes/checkYourAnswers.test.js

```diff
@@ -121,6 +121,48 @@ describe('GET /check-your-answers', () => {
       })
   })
 
+  it('Should contain prisoner compliant', () => {
+    reportService.getReportStatus.mockReturnValue({ complete: true })
+    reportService.getCurrentDraft.mockResolvedValue({
+      form: {
+        incidentDetails: {},
+        relocationAndInjuries: {
+          prisonerCompliant: true,
+        },
+      },
+    })
+
+    return request(app)
+      .get('/report/-35/check-your-answers')
+      .expect(200)
+      .expect('Content-Type', /html/)
+      .expect(res => {
+        expect(res.text).toContain('Was the prisoner compliant?')
+        expect(res.text).toContain('Yes')
```

Comment:
these tests are generally very good but this specific line is a bit off... its a big page and you can't really say that the `yes` is tied to that question!

Reply:
hmm.
I wanted to be able to test for the 'yes' if it was the only question on the page. 
So I have done this for the other sections
        incidentDetails: {},
        useOfForceDetails: {},
        evidence:{},
and the relocationAndInjuries object only has the one question answered
??

Reply:
It's not very easy to test at the moment.. some test framework provide css selector style ways of targeting elements. You could install [cheerio](https://github.com/cheeriojs/cheerio) as a dev dependency and try that if you want?

Reply:
ok. I 'll remove it for the time being.